### ISSUE TITLE
[OpenVINO] Enable Native Image Segmentation C++ Demo

### ISSUE THREAD

**Issue by mingmingtasd** [intel/webml-polyfill#1218]
The source code of this [Image Segmentation C++ Demo](https://docs.openvinotoolkit.org/latest/_demos_segmentation_demo_README.html) we have enabled blongs to [openvino-toolkit](https://software.intel.com/en-us/openvino-toolkit/choose-download), and it seems that the source code can't be found on github. While I find another Image Segmentation C++ Demo on https://github.com/opencv/open_model_zoo/tree/master/demos/segmentation_demo with similar source code and feature.

For the [Image Segmentation C++ Demo](https://docs.openvinotoolkit.org/latest/_demos_segmentation_demo_README.html) we have enabled, excute this command
 `./segmentation_demo -i woman.bmp -m /home/xu/Desktop/deeplabv3_mnv2/without_argmax/deeplabv3_mnv2_224/deeplabv3_mnv2_224.xml`, 
we can segment the first image to get the second image as output:
![woman](https://user-images.githubusercontent.com/52519066/79551283-0a237100-80cc-11ea-90e4-1d38e5a7b52f.jpg)
![out_0](https://user-images.githubusercontent.com/52519066/79551287-0abc0780-80cc-11ea-9edb-3c27637f99f7.jpg)

**Comment by fujunwei** [comment]
How to get the deeplabv3 openvino model? do you have any patches on the[ Image Segmentation C++ Demo ](https://docs.openvinotoolkit.org/latest/_demos_segmentation_demo_README.html)?

**Comment by mingmingtasd** [comment]
@fujunwei We converted tensorflow model with openvino tool to get this openvino model. The demo has been upgraded in openvino 2020.2 version and should to be rebuilt if needed.

**Comment by fujunwei** [comment]
Could you please share the patch you apply to the Demo? i want to verify it with old veriosn.

**Comment by mingmingtasd** [comment]
OK, I have sent to you privately. @fujunwei

**Comment by mingmingtasd** [comment]
On openvino 2020.3.194, the Native Image Segmentation C++ Demo has been updated. I tried to re-enable this demo to validate the deeplabv3 model (513*513) on ICL. I find that the updated demo code uses openCV to post-process output data but it doesn't  do pre-processing for input. So add some codes to resize the input picture, for example:
```
cv::Mat res_inImg;
cv::Size sz;
sz.width = 513;
sz.height = 513;
cv::resize(inImg, res_inImg, sz);
......
```
In addition, must set `-delay 0` in the command line as below:
`./segmentation_demo -m /home/webnn/Desktop/quantize/deeplabv3_DefaultQuantization/2020-06-22_11-00-05/optimized/deeplabv3.xml -i ~/Downloads/woman.jpg -delay 0`

The original deeplabv3 model and the quantized deeplabv3 model can both pass with the demo, check the details as below:
![Screenshot from 2020-07-02 13-07-46](https://user-images.githubusercontent.com/52519066/86320066-945d8900-bc68-11ea-847a-4baaa146d939.png)
![Screenshot from 2020-07-02 13-09-25](https://user-images.githubusercontent.com/52519066/86320072-97587980-bc68-11ea-9a5a-ce7ab57ae6a3.png)

@ibelem   @huningxin
